sources/it/regions.hjson

```diff
@@ -0,0 +1,180 @@
+  note: Italy regions
+  data: https://sophox.org/regions/geojson.json
+  attribution: [
+    {
+      label: {
+        en: © OpenStreetMap contributors
+      }
+      url: {
+        en: https://www.openstreetmap.org/copyright
+      }
+    }
+    {
+      label: {
+        en: Elastic Maps Service
+      }
+      url: {
+        en: https://www.elastic.co/elastic-maps-service
+      }
+    }
+  ]
+  fieldMapping: [
+    {
+      type: id
+      name: iso_3166_2
+      desc: ISO-3166-2 identifier
+    }
+    {
+      type: id
+      name: istat
+      desc: ISTAT ID
+    }
+    {
+      type: property
+      name: label_en
+      desc: Province name (English)
+    }
+    {
+      type: property
+      name: label_it
+      desc: Province name (Italian)
+    }
+  ]
+  query: {
+    sparql:
+    '''
+    #defaultView:MapRegions
+    SELECT ?id ?iso_3166_2
+            ?label_en ?label_it
+            ?istat_id
+    WHERE {
+      SERVICE <https://query.wikidata.org/sparql> {
+
+        SELECT ?id
+                (SAMPLE(?iso_3166_2) as ?iso_3166_2)
+                (SAMPLE(?label_en) as ?label_en)
+                (SAMPLE(?label_it) as ?label_it)
+                (SAMPLE(?istat_id) as ?istat_id)
+        WHERE {
+            # List of regions, whose sub-regions we want.
+            VALUES ?entity { wd:Q38 }
+
+            # P150 = "contains administrative territorial entity"
+            ?entity wdt:P150 ?id .
+            ?id wdt:P300 ?iso_3166_2 ;
+                wdt:P635 ?istat_id .
+
+            # Labels
+            OPTIONAL { ?id rdfs:label ?label_en . FILTER(LANG(?label_en) = "en") }
+            OPTIONAL { ?id rdfs:label ?label_it . FILTER(LANG(?label_it) = "it") }
+        }
+        # remove possible ID duplicates
+        GROUP BY ?id
+      }
+    }
+    ORDER BY ?iso_3166_2
+    '''
+  }
+  name: italy_regions
+  legacyIds: [
+    Italy Regions
+  ]
+  humanReadableName: {
+    af: Streke van Italië
+    an: Rechions d'Italia
+    ar: منطقة في إيطاليا
+    ast: rexón d'Italia
+    az: İtaliya regionları
+    bcl: Rehiyon kan Italya
+    be: рэгіён Італіі
+    be-tarask: вобласьць Італіі
+    bg: Административно деление на Италия
+    bn: ইতালীর প্রদেশসমূহ
+    br: Rannvroioù Italia
+    bs: Italijanske regije
+    ca: regió d'Itàlia
+    co: Regioni taliane
+    cs: italské regiony
+    cy: Rhanbarthau'r Eidal
+    da: Italiens regioner
+    de: Region in Italien
+    diq: Mıntıqeyê İtalya
+    el: Περιφέρειες της Ιταλίας
+    en: Italy Regions
+    eo: regiono de Italio
+    es: Región de Italia
+    et: Itaalia maakonnad
+    eu: Italiako eskualdea
+    fa: "فهرست ناحیه\u200cهای ایتالیا"
+    fi: Italian alueet
+    fr: région de l'Italie
+    fur: Regjons de Italie
+    fy: Italjaanske Regio’s
+    gsw: Italienische Regionen
+    he: מחוזות איטליה
+    hr: Talijanske regije
+    hu: Olaszország régiói
+    ia: region de Italia
+    id: Daftar wilayah di Italia
+    is: Héruð Ítalíu
+    it: regione d'Italia
```

Comment:
Where is used this field? Because `regione d'italia` is singular and is not the correct translation for `Italian regions`

Reply:
If or when Kibana is localized to Italian, this appears as the layer name in the select list. Is the correct plural form `regioni d'Italia`?

Reply:
both `Regioni Italiane` or `Regioni d'Italia`  sounds fine for me

Reply:
I'll fix Spanish and Catalan for plurals and wait for @markov00 on Italian, thanks for your help!

Reply:
Let's go with `Regioni d'Italia`
https://it.wikipedia.org/wiki/Regioni_d%27Italia